engine/src/main/java/org/terasology/engine/utilities/OS.java

```diff
@@ -13,14 +15,23 @@
     MACOSX,
     LINUX;
 
+    public static final String OS_NAME = System.getProperty("os.name").toLowerCase();
+    public static final String ARCHITECTURE = System.getProperty("os.arch");
+    public static final String USER_DIRECTORY = System.getProperty("user.dir");
+    public static final String PATH_SEPARATOR = FileSystems.getDefault().getSeparator();
+
+    public static final boolean IS_WINDOWS = OS_NAME.toLowerCase().contains("win");
+    public static final boolean IS_MACOSX =  OS_NAME.toLowerCase().contains("mac");
+    public static final boolean IS_LINUX =  OS_NAME.toLowerCase().contains("linux") | OS_NAME.toLowerCase().contains("freebsd") || OS_NAME.toLowerCase().contains("sunos") || OS_NAME.toLowerCase().contains("unix");
+    public static final boolean IS_64 = ARCHITECTURE.endsWith("64");
```

Comment:
You changed the logic here to be the one from `NativeHelper` - are we sure this works the same as the previous implementation in this file? The [Platform class from the Launcher](https://github.com/MovingBlocks/TerasologyLauncher/blob/master/src/main/java/org/terasology/launcher/util/Platform.java) uses yet another logic to determine things 🙈 

Is there a best practice way to do it out there?